modules/wpml/wpml-api.php

```diff
@@ -28,7 +28,7 @@ public function __construct() {
 		// Site Wide Language informations
 
 		add_filter( 'wpml_active_languages', array( $this, 'wpml_active_languages' ), 10, 2 );
-		add_filter( 'wpml_display_language_names', array( $this, 'wpml_display_language_names' ), 10, 5 );
+		add_filter( 'wpml_display_language_names', array( $this, 'wpml_display_language_names' ), 10, 2 );
```

Comment:
```suggestion
		add_filter( 'wpml_display_language_names', array( $this, 'wpml_display_language_names' ), 10, 2 ); // Because we don't translate language names, 3rd to 5th parameters are not supported.
```